packages/devextreme/js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -144,9 +146,32 @@ const TextEditorMask = TextEditorBase.inherit({
 
     _onMouseWheel: noop,
 
-    _render: function() {
+    _useMaskBehavior() {
+        return Boolean(this.option('mask'));
+    },
+
+    _attachDragEventHandlers() {
+        const useMaskBehavior = this._useMaskBehavior();
+
+        if(!useMaskBehavior) {
+            return;
+        }
+
+        const eventName = addNamespace(DROP_EVENT_NAME, this.NAME);
+        const input = this._input();
+
+        const callback = (e) => {
```

Comment:
[code style]
This `callback` as a separate variable does not improve clearness. U can get rid of this variable or rename it, for example `dropHandler`

Reply:
I have used the function for debugging. Now I would rather delete a separate variable. Fixed in [f0b0bf5f2a89e12ddc6f0d76ad534fe221a08aef].